packages/@lwc/engine-core/src/framework/hooks.ts

```diff
@@ -34,6 +64,263 @@ import { applyEventListeners } from './modules/events';
 import { applyStaticClassAttribute } from './modules/static-class-attr';
 import { applyStaticStyleAttribute } from './modules/static-style-attr';
 
+const enum LwcDomMode {
```

Comment:
same q here, is there any functional change in this block other than just copying it from the api.ts file?

Reply:
Same observation. No change was made to this routine.